src/scene/vcMedia.cpp

```diff
@@ -87,13 +91,13 @@ void vcMedia::OnNodeUpdate(vcState *pProgramState)
   if (m_pNode->pURI == nullptr)
   {
     m_loadStatus = vcSLS_Failed;
-
-    if (m_image.pTexture != nullptr)
-      vcTexture_Destroy(&m_image.pTexture);
+    vcTexture_Destroy(&m_image.pTexture);
   }
-
-  if (m_loadStatus == vcSLS_Loaded && ((m_pNode->pURI != nullptr && !udStrEqual(m_pLoadedURI, m_pNode->pURI)) || (m_reloadTimeSecs != 0.0 && m_loadLoadTimeSec + m_reloadTimeSecs < udGetEpochSecsUTCf())))
+  else if ((m_loadStatus == vcSLS_Loaded && !udStrEqual(m_pLoadedURI, m_pNode->pURI)) ||
+    (m_reloadTimeSecs != 0.0 && m_loadLoadTimeSec + m_reloadTimeSecs < udGetEpochSecsUTCf()) ||
+    (m_loadStatus == vcSLS_Unloaded))
```

Comment:
New lines aren't required here, if you want to split it out, bool variables are preferred